services/discordService.js

```diff
@@ -52,7 +52,31 @@ const setInDiscordFalseScript = async () => {
   await Promise.all(updateUsersPromises);
 };
 
+const generateAuthTokenForCloudflare = async () => {
```

Comment:
- Why is this inside `services` and not `utils` ?

Reply:
Made the changes